### PR TITLE
Adding py2vega recipe

### PR DESCRIPTION
**Pull request by martinRenou**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by martinRenou** [comment]
@SylvainCorlay Could you merge this?

File: recipes/py2vega/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 370d27e12d4add32a6b6092c347905d716d18687589bc59c0b90b76fdf79619b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
```
Comment (marcelotrevisani) [inline recipes/py2vega/meta.yaml]: I think this recipe can be ``noarch``
```suggestion
  noarch: python
```

and after that you can select the python versions which you want to

File: recipes/py2vega/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/py2vega/meta.yaml]:
```suggestion
    - python >3.5
```

File: recipes/py2vega/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    - pip
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/py2vega/meta.yaml]:
```suggestion
    - python >3.5
```

**Comment by martinRenou** [comment]
Thanks for the feedback @marcelotrevisani !

**Comment by martinRenou** [comment]
Thanks !

**Pull request merged**